esss_fix_format/cli.py

```diff
@@ -119,7 +85,17 @@ def main(files_or_directories, check, stdin, commit):
         extension = os.path.normcase(os.path.splitext(filename)[1])
 
         if extension == '.py':
-            sorter = isort.SortImports(file_contents=new_contents, **ISORT_CONFIG)
+            settings_path = os.path.dirname(filename)
+            settings_loaded = isort.settings.from_path(settings_path)
```

Comment:
I think this will cause the project's `.isort.cfg` to be loaded (not the esss_fix_format one).

Reply:
That's actually the idea... After talking to @nicoddemus we thought this was a better approach (so, we'll add .isort.cfg files to our repos) to integrate better with other standalone tools (as .isort.cfg is always searched in that path).

It'd be possible to have each use configure it, but in the end we agreed adding it to the repo was less trouble that having everyone configure it explicitly.